src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class Lottery extends Ball {
+    public String getRandomBall() {

```

Comment:
```suggestion
    public Ball getRandomBall() {

```

метод getRandomBall() должен возвращать объект класса Ball